config.py

```diff
@@ -86,7 +86,7 @@ class BaseConfig(object):
     )
 
     DEBUG = False
-    ERROR_404_HELP = False
+    RESTX_ERROR_404_HELP = False
```

Comment:
This was a warning on EVERY SINGLE TEST due to deprectaion and I got irritated enough to fix it